packages/standard-components/src/forms/InnerForm.svelte

```diff
@@ -108,8 +116,28 @@
 
       return !newError
     }
+
+    const clearValue = () => {
+      const { fieldState } = fieldMap[field]
```

Comment:
I would maybe declare a new variable here for the value we're going to revert to, to save duplication down below. We need to check if there's an initial value here, so something like:
`const newValue = initialValues[field] ?? defaultValue`